semcore/notice/src/Notice.jsx

```diff
@@ -50,6 +50,7 @@ class RootNotice extends Component {
         use:theme={useTheme}
         backgroundColor={color}
         role="alert"
+        aria-live="assertive"
```

Comment:
it seems to me that `aria-live="assertive"` can only be used for the error type.
And for the rest it should be `aria-live="polite"`.
If so, then the alert role is not needed.